src/main/kotlin/no/nav/klage/oppgave/api/view/OversendtKlage.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.klage.oppgave.api.view
+
+import no.nav.klage.oppgave.domain.klage.Mottak
+import no.nav.klage.oppgave.domain.kodeverk.Kilde
+import no.nav.klage.oppgave.domain.kodeverk.Sakstype
+import no.nav.klage.oppgave.domain.kodeverk.Tema
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+import javax.validation.constraints.Past
+import javax.validation.constraints.Pattern
+
+data class OversendtKlage(
+    val uuid: UUID,
+    val tema: Tema,
```

Comment:
Når du har Tema her, og ikke "tema-id", hva må man da sende inn? Kotlin-enum-navnet? Vi ønsker vel at de sender inn IDen på Temaet egentlig? Jeg hadde dette problemet i outputen til KodeverkControlleren. Mulig det kunne vært løst ved noe jackson-magi, men jeg undersøkte ikke det da..

Reply:
Hvis vi bruker Tema så forventer vi vel "AAP", "FOR" osv? Det er vel det vi må støtte siden det er koeverket